### PR TITLE
chore(docs): refactor readme

### PR DESCRIPTION
**Pull request by 3arthqu4ke**
Release-please seems to update all version strings within a markdown code block.

### CODE REVIEW

**Comment by 3arthqu4ke** [comment]
@ChipWolf  Ok one second: this was done by renovate automatically, so there's a problem with that.
https://github.com/headlesshq/mc-runtime-test/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R82

**Comment by ChipWolf** [comment]
Oh hah it's the comments in the markdown file

**Comment by 3arthqu4ke** [comment]
I see, same issue as this: https://github.com/googleapis/release-please/issues/2345
It sadly does not seem possible to define an own VersionUpdater really.

We could define the code block with html like this
```html
<!-- x-release-please-start-version -->
<pre>
<code class="language-yml">
... yaml stuff
```
But the yaml syntax highlighting wont work.

I think I will just update the version string manually within the README now?

File: README.md [hunk 1]
```diff
@@ -67,7 +66,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install Java
-        uses: useblacksmith/setup-java@v5
```
Comment (ChipWolf) [inline README.md:70]: This should be `useblacksmith/setup-java@v5` for our custom runners

File: README.md [hunk 2]
```diff
@@ -79,13 +78,12 @@ jobs:
       - name: Run the MC client
         uses: headlesshq/mc-runtime-test@3.0.0
         with:
-          mc: 3.0.0
+          mc: 1.21.4
```
Comment (ChipWolf) [inline README.md:81]: `mc: 1.21.4 # x-release-please-version`

File: release-please-config.json [hunk 3]
```diff
@@ -9,8 +9,7 @@
       "release-type": "simple",
       "extra-files": [
         "action.yml",
-        "api/gradle.properties",
-        "README.md"
```
Comment (ChipWolf) [inline release-please-config.json:13]: We should continue to update the README automatically based on my other suggested change

**Review by ChipWolf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (3arthqu4ke) [reply README.md:70]: My thought process was, that the average user would be more familiar with the actions/setup-java@v4 you see everywhere. We could mention blacksmith as a faster alternative, but the README should show the fastest way to this up.

File: README.md (hunk 2 above)
Comment (3arthqu4ke) [reply README.md:81]: hmmmmmm... yeah Ill think about it later

Comment (3arthqu4ke) [reply README.md:81]: I still really dislike this because it looks like a comment, which could confuse readers.

**Comment by 3arthqu4ke** [comment]
I think I will try a workflow that runs after release-please and fixes the readme

**Comment by ChipWolf** [comment]
> I think I will try a workflow that runs after release-please and fixes the readme

```bash
sed -i -E 's/\smc: .*/mc: 1.21.4/' README.md
```

**Comment by ChipWolf** [comment]
@3arthqu4ke I've put a good amount of thought into this

In theory, we should only have the major version referenced in the README, i.e. `headlesshq/mc-runtime-test@v3`, as most users don't keep their actions updated at any reasonable cadence.

To achieve this we're going to have to do some trickery in the release pipeline, I'll be setting up a PR for this soon

We should also assess which tags are in use and push deprecated notices to them all.

File: README.md (hunk 1 above)
Comment (ChipWolf) [reply README.md]: ohhh I just spotted this was in the readme

**Review by ChipWolf — APPROVED** [review]
LGTM

**Pull request merged**